ctapipe/reco/hillas_intersection.py

```diff
@@ -82,7 +82,7 @@ def predict(self, hillas_parameters, tel_x, tel_y, array_direction):
         sky_pos = SkyCoord(
             az=src_x * u.deg,
             alt=src_y * u.deg,
-            frame=HorizonFrame()
+            frame=HorizonFrame(location=array_direction.location, obstime=array_direction.obstime)
```

Comment:
Isn't it then `SkyCoord(az=..., alt=..., frame=array_direction.frame)`?

Reply:
yes! in this way all the information in the should be passed.